### PR TITLE
feat(config/giphy) add proxyUrl config for giphy requests

### PR DESCRIPTION
**Pull request by mihhu**

### CODE REVIEW

File: react/features/gifs/components/native/GifsMenu.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
+import { setServerUrl } from '@giphy/js-fetch-api';
```
Comment (saghul) [inline react/features/gifs/components/native/GifsMenu.js:1]: Is this also used by the RN SDK? AFAIK it uses their native SDKs under the hood.

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "@atlaskit/tooltip": "17.1.2",
     "@emotion/react": "11.10.0",
     "@emotion/styled": "11.10.0",
-    "@giphy/js-fetch-api": "4.1.2",
+    "@giphy/js-fetch-api": "^4.7.1",
```
[the remark this answers is not in the document]
Comment (mihhu) [reply package.json]: > Also, don't we need to update any other package?

You mean in general or related to this pr? If it's the latter, then I don't think so.

[the remark this answers is not in the document]
Comment (saghul) [reply package.json]: Related to this PR.

**Review by saghul — APPROVED** [review]
LGTM if the testing went well.

**Pull request merged**